xlators/features/cloudsync/src/cloudsync.c

```diff
@@ -306,9 +308,13 @@ cs_readdirp(call_frame_t *frame, xlator_t *this, fd_t *fd, size_t size,
 
     STACK_WIND(frame, default_readdirp_cbk, FIRST_CHILD(this),
                FIRST_CHILD(this)->fops->readdirp, fd, size, off, xdata);
+    if (xref)
```

Comment:
Our coding standard says that curly brackets need to be added always, even if there's a single line in the code block.

Reply:
Ack